src/style.rs

```diff
@@ -307,6 +307,24 @@ impl StyleBuilder {
         Self::default()
     }
 
+    /// Derives relevant fields from `egui::Style` and sets the remaining fields to their default values.
+    ///
+    /// Fields overwritten by [`egui::Style`] are:
```

Comment:
Replace this list with `See also: [link to Style::from_egui]`.

The reason for that is every time some new affected fields are added, the documentation would need to be updated in two places, which is easy to forget about and we don't want that.